content/docs/user-guide/remote-objects.md

```diff
@@ -0,0 +1,119 @@
+# Working with Remote Objects and Projects
+
+[MLEM objects] (models, data, etc.) can live in different locations such as Git
+repositories, cloud object storage, local directories, etc.
+
+<admon type="note">
+
+In this page we will work with model objects in Git repos, but the same
+operations apply to any [object type] and location.
+
+</admon>
+
+[mlem objects]: /doc/user-guide/basic-concepts#mlem-objects
+[object type]: /doc/user-guide/basic-concepts#mlem-object-types
+
+## Listing objects
+
+You can list MLEM objects inside a remote MLEM project in a Git repo with
+`mlem list`. There's no need to clone the repo.
+
+```cli
+$ mlem list \
+       https://github.com/iterative/example-mlem-get-started \
+       --rev simple
```

Comment:
Another question: does `mlem list` work on non-Git MLEM projects?

Reply:
@jorgeorpinel, Yes, it does!

Reply:
Well then this needs a quick update:

````suggestion
## Listing objects

You can list MLEM objects inside a remote MLEM project (e.g. in a Git
repo) with `mlem list`. There's no need to download/clone the project.

```cli
$ mlem list \
       https://github.com/iterative/example-mlem-get-started \
       --rev simple
````